File: SDKs/Aspose.Email_Cloud_SDK_for_Java1/src/main/java/com/aspose/email/model/EmailProperties.java
package com.aspose.email.model;

import java.util.*;
import com.aspose.email.model.Link;
public class EmailProperties {
  private Link Link = null;
  private List<EmailProperty> List = new ArrayList<EmailProperty>();
  /**
	 * getLink
	 * Gets Link
	 * @return Link
	 */
  public Link getLink() {
    return Link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param Link Link
	 */
  public void setLink(Link Link) {
    this.Link = Link;
  }

  /**
	 * getList
	 * Gets List<EmailProperty>
	 * @return List
	 */
  public List<EmailProperty> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<EmailProperty>
	 * @param List List<EmailProperty>
	 */
  public void setList(List<EmailProperty> List) {
    this.List = List;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class EmailProperties {\n");
    sb.append("  Link: ").append(Link).append("\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Email_Cloud_SDK_for_CSharp/src/Com/Aspose/Email/Model/EmailProperties.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Email.Model {
  public class EmailProperties {
    public Link Link { get; set; }

    public List<EmailProperty> List { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class EmailProperties {\n");
      sb.Append("  Link: ").Append(Link).Append("\n");
      sb.Append("  List: ").Append(List).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
